config/config/wc-config.yaml

```diff
@@ -104,10 +104,10 @@ opa:
     URL:
       - set-me
       - harbor.example.com
+      - quay.io/jetstack/cert-manager-acmesolver
       ## Uncomment this if you want to add an OPA exception for cert-manager-acmesolver.
       ## This should be added only as an exception, e.g: if you cannot rely on a harbor instance managed by users.
       ## Otherwise set the correct repository for this image in the certmanager section.
```

Comment:
Either remove this comment or move it above the new line and rewrite it a little to match the fact that it is on by default.

Reply:
Thanks Viktor , i will remove these lines as it is meaningless now after adding it as a default.

Reply:
Done now :)